ui/screens/levelselect.lua

```diff
@@ -109,9 +115,16 @@ local function make_level_element(pack, level, extra_info)
             if level_element_selected ~= self then
                 local elems = self.parent.elements
                 for i = 1, #elems do
-                    elems[i].background_color = { 0, 0, 0, 0.7 }
+					if(elems[i] == self) then 
+						description = label:new(level.description, { font_size = 16, wrap = true })
+						self.background_color = { 0.5, 0.5, 0, 0.7 }
+					else
+						description = label:new("", { font_size = 16, wrap = true })
+						elems[i].background_color = { 0, 0, 0, 0.7 }
+					end
+					elems[i].element.elements[2].elements[2] = update_element(description, elems[i].element.elements[2], 2, elems[i].element.elements[2].elements[2])
                 end
-                self.background_color = { 0.5, 0.5, 0, 0.7 }
+                root.elements[2] = update_element(root.elements[2], root, 2, root.elements[2])
```

Comment:
We'll probably replace this update_element function at some point as it is kind of messy, but for now this if fine ig.